### PR TITLE
sort remarks for n*log(n) printing

### PR DESCRIPTION
**Pull request by oscardssmith**
fixes https://github.com/JuliaDebug/Cthulhu.jl/issues/259

### CODE REVIEW

**Comment by oscardssmith** [comment]
I don't think the failures on nightly are my fault, but I'm not 100% sure.

**Comment by aviatesk** [comment]
> One question: Is there 1 remark per pc? If not, this is slightly broken and it needs to iterate a bit.

yes there can be multiple remarks per statement

File: src/codeview.jl [hunk 1]
```diff
@@ -167,14 +167,15 @@ function cthulhu_typed(io::IO, debuginfo::Symbol,
...
         function postprinter(io::IO, @nospecialize(typ), used::Bool)
             _postprinter(io, typ, used)
             haskey(io, :idx) || return
             idx = io[:idx]::Int
-            for (pc, remark) in remarks
-                if pc == idx
-                    printstyled(io, ' ', remark; color=:light_black)
-                end
+            firstind = searchsortedfirst(remarks, idx=>"")
+            for i in firstind:lastindex(remarks)
```
Comment (aviatesk) [inline src/codeview.jl:176]: can we further limit this iteration space to `firstind:searchsortedfirst(remarks, idx+1=>"")`?

Comment (aviatesk) [reply src/codeview.jl:176]: ah I see.

**Pull request merged**